GitUI/Resources/ChangeLog.md

```diff
@@ -2,225 +2,225 @@
 Changelog
 =========
 ### Version 2.49.03 (26 March 2017)
-* Fixed issue #3605. NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
-* Fixed issue #3578. File history and blame not show when path to file contain cyrilic chars bug reproducible.
+* Fixed issue [#3605](//github.com/gitextensions/gitextensions/issues/3605). NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
```

Comment:
should it be `[#3605](https://github.com/gitextensions/gitextensions/issues/3605)`?